pom.xml

```diff
@@ -69,6 +73,18 @@
       <version>${junit.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
+      <version>${junit.vintage.version}</version>
+      <scope>runtime</scope> <!-- test-runtime -->
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId> <!-- junit-jupiter-api, Surefire is not smart enough, yet. -->
```

Comment:
Blech. They say one of Jupiter’s benefits is better integration with test runners. But I see that it isn’t integrated with Maven yet. That doesn’t inspire confidence.

Reply:
> Blech.

Hey, I understand that. (-:

Come over and help improving Surefire!

Or try https://github.com/sormuras/junit-platform-maven-plugin which already "is smart enough" to add required runtime dependencies.

Reply:
Surefire 2.22.2 or the upcoming 3.0.0-M4 do help in this configuration ... "Blech". I'll upgrade the PR ... soon.